packages/react/src/utils/index.tsx

```diff
@@ -0,0 +1,21 @@
+import isNumeric from './isNumeric'
```

Comment:
Just wanted to check-in, are the exported utilities in this file ones that currently being used externally? If possible, it'd be great to limit what we export publicly to reduce our surface area.